### PR TITLE
Adding git bash support, closes #94

### LINKED ISSUE #94: Mingw support?
Will there any potential future support for mingw-w64 go/windows native (not wsl)?
I'm not exactly sure how sshcode and go work so I'm sorry if this is a dumb question

### PR DESCRIPTION
**Pull request by eargollo**
- Adding support to Windows when using git bash
- It is able to understand relative paths on Windows and translate to the Linux server
- In order to work, user will need to download rsych for Windows and put on path

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -112,3 +121,24 @@ Arguments:
...
+	}
+	rel, err := filepath.Rel(usr.HomeDir, dir)
+	if err != nil {
+		return dir
+	}
+	rel = "~/" + filepath.ToSlash(rel)
+	return rel
+}
+
+func gitbashWindowsDir(dir string) (res string) {
```
Comment (deansheather) [inline main.go]: Could you add a comment stating why this is required? Something along the lines of:
```
// gitbashWindowsDir translates a directory similar to `C:\Users\username\path` to `~/path` for compatibility with Git bash.
```

**Comment by eargollo** [comment]
@deansheather, I fixed all the issues but the enable connection error you mentioned. I could not reproduce it. Can you give me some more info on it?

**Comment by Merith-TK** [comment]
My build is finnished for now. I am waiting for @eargollo to do his thing.

**Comment by Merith-TK** [comment]
@eargollo Can you email me with the email on my profile?

**Comment by eargollo** [comment]
@Merith-TK, sorry for the long delay. I was on vacations.

What is the status? How can I be of help?

**Comment by Merith-TK** [comment]
No worries! I got the issues with your code fixed. Look at the new PR for
the current issues

**Comment by deansheather** [comment]
Should this PR be closed in favor of the other one? @Merith-TK @eargollo

**Comment by Merith-TK** [comment]
I think so, but according to my last communication with @eargollo , he is mostlikely swamped by work. 

Currently my fork is working with oen caviat the i cant explain... I sent it to you over discord @deansheather

**Pull request closed without merging**